index.html

```diff
@@ -263,6 +263,8 @@ <h2># Contact</h2>
 
       We are also happy to welcome volunteers to prepare the unconf or on the unconf itself. Most of preparation tasks
       can be done remotely. Just contact us!
+
```

Comment:
it's pure HTML, so there may be some `<br />`'s instead of just linebreaks?

Reply:
Right, thanks. I fixed it

Reply:
![Screenshot 2020-02-06 at 11 34 00](https://user-images.githubusercontent.com/52921025/73929477-238e8c80-48d5-11ea-9158-32f996da1f7f.png)